src/workflows/airqo_etl_utils/ml_utils.py

```
@@ -786,30 +999,116 @@ def process_faulty_devices_fault_sequence(df: pd.DataFrame):
+            merged_df["anomaly_percentage_fault"] = (
+                merged_df["anomaly_percentage"] > 45
+            ).astype(int)
+        if "fault_count" in merged_df.columns and "anomaly_sequence_fault" not in merged_df.columns:
+            merged_df["anomaly_sequence_fault"] = (
+                merged_df["fault_count"] >= 80
+            ).astype(int)
+        fault_flag_columns = [
+            column
+            for column in (
+                FaultDetectionUtils.RULE_FAULT_COLUMNS
+                + ["anomaly_percentage_fault", "anomaly_sequence_fault"]
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)